claasp/cipher_modules/models/milp/utils/milp_name_mappings.py

```diff
@@ -14,4 +14,5 @@
 MILP_BUILDING_MESSAGE = "Building model in progress ..."
 MILP_XOR_DIFFERENTIAL_OBJECTIVE = "probability"
 MILP_XOR_LINEAR_OBJECTIVE = "probability"
-MILP_TRUNCATED_XOR_DIFFERENTIAL_OBJECTIVE = "number_of_unknown_patterns"
\ No newline at end of file
+MILP_TRUNCATED_XOR_DIFFERENTIAL_OBJECTIVE = "number_of_unknown_patterns"
+MILP_WEIGHT_PRECISION = 100
```

Comment:
pass the weight precision as a parameter with 100 as default, so that a user can change it without changing the source code